custom-assembly-report-generator/generate_custom_assembly_report.py

```diff
@@ -0,0 +1,184 @@
+        print("INFO: No entries were found in the assembly report with non-equivalent GenBank accessions")
+    for accession in refseq_accessions_with_non_equivalent_genbank_accessions:
+        equivalent_genbank_accession = \
+            genbank_equivalents_dataframe[genbank_equivalents_dataframe["rs_acc"] == accession]["gb_acc"]
+        if len(equivalent_genbank_accession) == 0:
+            print("WARN: Could not find equivalent GenBank accession for RefSeq accession " + accession)
+        else:
+            equivalent_genbank_accession = equivalent_genbank_accession.values[0]
+            assembly_report_dataframe.loc[assembly_report_dataframe["RefSeq-Accn"] == accession, "GenBank-Accn"], \
+            assembly_report_dataframe.loc[assembly_report_dataframe["RefSeq-Accn"] == accession, "Relationship"] \
+                = equivalent_genbank_accession, "="
+    return assembly_report_dataframe
+
+
+def update_non_equivalent_refseq_accessions_in_assembly_report(assembly_report_dataframe,
+                                                               genbank_equivalents_dataframe):
+    genbank_accessions_with_non_equivalent_refseq_accessions = \
+        get_genbank_accessions_without_equivalent_refseq_accessions(assembly_report_dataframe)
+    if len(genbank_accessions_with_non_equivalent_refseq_accessions) == 0:
+        print("INFO: No entries were found in the assembly report with non-equivalent RefSeq accessions")
+    for accession in genbank_accessions_with_non_equivalent_refseq_accessions:
+        equivalent_refseq_accession = \
+            genbank_equivalents_dataframe[genbank_equivalents_dataframe["gb_acc"] == accession]["rs_acc"]
+        if len(equivalent_refseq_accession) == 0:
+            print("WARN: Could not find equivalent RefSeq accession for GenBank accession " + accession)
+        else:
+            equivalent_refseq_accession = equivalent_refseq_accession.values[0]
+            assembly_report_dataframe.loc[assembly_report_dataframe["GenBank-Accn"] == accession, "RefSeq-Accn"], \
+            assembly_report_dataframe.loc[assembly_report_dataframe["GenBank-Accn"] == accession, "Relationship"] \
+                = equivalent_refseq_accession, "="
+    return assembly_report_dataframe
+
+
+def main(metadb, metauser, metahost, species, assembly_accession, genbank_equivalents_file):
+    for species_db_info in data_ops.get_species_pg_conn_info(metadb, metauser, metahost):
+        if species_db_info["database_name"] == species:
+            # Download assembly report
+            assembly_report_path = download_assembly_report(assembly_accession)
+
+            # Get dataframes by reading in the assembly report and the file with GenBank equivalents
+            assembly_report_dataframe, genbank_equivalents_dataframe = \
+                get_dataframe_for_assembly_report(assembly_report_path), \
+                get_dataframe_for_genbank_equivalents(genbank_equivalents_file, assembly_accession)

```

Comment:
Split these declaration in 2 lines to increase readability. Given they are 2 completely separate calls, there is no benefit in grouping them.